packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -41,6 +41,7 @@ export function run() {
       !!process.env.JUPITERONE_DEV,
     )
     .option('--api-base-url <url>', 'API base URL used during run operation.')
+    .option('-V, --disable-schema-validation', 'disable schema validation')
```

Comment:
Do we use `-V` as the short hand for this in other places?

Reply:
We do.  I had it match the option used in `j1-integration collect`